### PR TITLE
update interceptor, faces, websocket API jars for TCK builds.

### PR DESCRIPTION
**Pull request by gurunrao**
update interceptor, faces, websocket API jars for TCK builds.
Signed-off-by: gurunandan.rao@oracle.com <gurunandan.rao@oracle.com>

### CODE REVIEW

File: docker/pom.xml [hunk 1]
```diff
@@ -32,13 +32,13 @@
...
-        <jakarta.faces-api.version>3.0.0</jakarta.faces-api.version>
+        <jakarta.faces-api.version>4.0.0-M2</jakarta.faces-api.version>
         <!-- Jakarta WebSocket -->
-        <websocket-api.version>2.0.0</websocket-api.version>
+        <websocket-api.version>2.1.0</websocket-api.version>
         <!-- Jakarta Concurrency -->
         <concurrent-api.version>2.0.0</concurrent-api.version>
         <!-- Jakarta Interceptors -->
-        <jakarta.interceptor-api.version>2.1.0-RC1</jakarta.interceptor-api.version>
+        <jakarta.interceptor-api.version>2.1.0</jakarta.interceptor-api.version>
```
Comment (scottmarlow) [inline docker/pom.xml:41]: Where is interceptor 2.1.0 pushed to?  

I don't see interceptor 2.1 mentioned on https://jakarta.ee/specifications/interceptors/

I also don't see interceptor 2.1 mentioned on https://docs.google.com/spreadsheets/d/1YTUpfdLZZrk2_UGwoX2w0seOCueRO3sQJIjWxpDAa7g/edit?pli=1#gid=35969432

https://www.eclipse.org/lists/interceptors-dev/threads.html#00036 does have discussion about doing a service release of Interceptors but I don't believe that happened yet.

Comment (gurunrao) [reply docker/pom.xml]: there was interceptor 2.1.0 at staging,looks like as part of https://docs.google.com/document/d/1DF0T2hbJqDNsY0v_lYFKCz-ZgHPbnMUIu6mhmt5LCJU/edit?pli=1#, 2.1.0 got deleted, have used 2.1.0-RC1.

**Comment by scottmarlow** [comment]
Interceptor 2.1.0-RC1 artifact will be deleted and replaced I think, so this pr looks correct except the [2.1.0-RC1](https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/interceptor/jakarta.interceptor-api/2.1.0-RC1/) artifact still looks old:
```

jakarta.interceptor-api-2.1.0-RC1.jar | Tue Nov 16 19:35:39 UTC 2021 | 25217 



```

**Review by scottmarlow — APPROVED** [review]

**Comment by gurunrao** [comment]
> Interceptor 2.1.0-RC1 artifact will be deleted and replaced I think, so this pr looks correct except the [2.1.0-RC1](https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/interceptor/jakarta.interceptor-api/2.1.0-RC1/) artifact still looks old:
> 
> ```
> 
> jakarta.interceptor-api-2.1.0-RC1.jar | Tue Nov 16 19:35:39 UTC 2021 | 25217 
> ```

have made interceptor to use 2.0.0(which has latest timestamp).

**Pull request merged**